### PR TITLE
Nominate Henry Zhang who's the founder of Harbor project as core maintainer

### PR DESCRIPTION
**Pull request by steven-zou**
I'd like to nominate [hainingzhang](https://github.com/hainingzhang) who's the founder of Harbor project as a new core maintainer with high respects as he's still willing to supervise, promote and do contributions to the Harbor community!

Signed-off-by: Steven Zou <szou@vmware.com>

### CODE REVIEW

**Comment by kofj** [comment]
support

File: MAINTAINERS.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ describes governance guidelines and maintainer responsibilities.
 | Qian Deng | [ninjadq](https://github.com/ninjadq) | [VMware](https://www.github.com/vmware/) |
 | Mia Zhou | [zhoumeina](https://github.com/zhoumeina) | [VMware](https://www.github.com/vmware/) |
 | Steven Zou | [steven-zou](https://github.com/steven-zou) | [VMware](https://www.github.com/vmware/) |
+| Henry Zhang| [hainingzhang](https://github.com/orgs/goharbor/people/hainingzhang)| [VMware](https://www.github.com/vmware/) |
```
Comment (nlowe) [inline MAINTAINERS.md]: This link goes to Henry's page on the goharbor project, which isn't public to everyone. Let's link to his github profile instead:

```suggestion
| Henry Zhang| [hainingzhang](https://github.com//hainingzhang)| [VMware](https://www.github.com/vmware/) |
```

Comment (steven-zou) [reply MAINTAINERS.md]: I'll update it after PR approved later. Thank you @nlowe !

**Comment by steven-zou** [comment]
Base one the voting result (5/5 votes), Henry zhang's nomination is approved!

Screen Shot 2019-04-25 at 18 36 49

**Pull request merged**